Core/SiteRating.swift

```diff
@@ -40,11 +40,13 @@ public class SiteRating {
     }
     
     public var containsMajorTracker: Bool {
-        return trackersDetected.contains(where: { $0.key.fromMajorNetwork() } )
+        return trackersDetected.contains(where: { $0.key.parent != nil && MajorTrackerNetworks.networks.contains($0.key.parent!) })
     }
     
-    public func trackerDetected(_ tracker: Tracker, blocked: Bool) {
-        
+    public func trackerDetected(_ tracker: String, parent: String? = nil, blocked: Bool) {
```

Comment:
I'm working on the Site Rating, is it easy for you to back this api change out? I don't think you're calling this method so I'm hoping it will be ok but happy discuss if that's not the case. Feel free to keep the inner Tracker type if that makes thing easier, e.g
```public func trackerDetected(_ tracker: SiteRating.Tracker, blocked: Bool)```
I just want the overall signature to stay stable